models/PyCryptoBot.py

```diff
@@ -1023,50 +1034,86 @@ def startApp(self, account, last_action='', banner=True):
                 attempts = 0
 
                 if self.simstartdate is not None and self.simenddate is not None:
-                    date = self.simstartdate.split('-')
-                    startDate = datetime(int(date[0]), int(date[1]), int(date[2]))
+												   
+                    startDate = self.getDateFromISO8601Str(self.simstartdate.replace("T", " "))
+																				   
                     if self.simenddate == 'now':
-                        endDate = datetime.now()
+                        endDate = self.getDateFromISO8601Str(str(datetime.now()).split(".")[0])
```

Comment:
what is the logic behind this? instead of using `datetime.now()` there's quite complicated conversion that starts from `datetime.now()` and probably results in the same value? am I missing something here?

Reply:
This might work:
```
>>> now = datetime.datetime.now()
>>> now
datetime.datetime(2021, 7, 20, 9, 20, 19, 936057)
>>> now - datetime.timedelta( 0, 0, now.microsecond )
datetime.datetime(2021, 7, 20, 9, 20, 19)
```
In either case, a simple comment like "# round up to seconds" would be really helpful to the people who come after you =)